tetris.py

```diff
@@ -21,7 +21,7 @@
 TEAL = "paleturquoise3"
 WHITE = 'snow'
 
-block_letters = ["I", "L", "L_rev", "O", "E", "Z", "Z_rev"]
+block_letters = ["I"]
```

Comment:
Did you `git diff --cached` before commit? :)

Reply:
No, but this is not a big deal. Will do it next time though :)